doc/index.rst

```diff
@@ -1,58 +1,44 @@
-
-Autocrypt Python Reference tool and implementation
-==================================================
+Autocrypt support for mail agents
+=================================
 
 .. note::
 
-    The py-autocrypt tool is as much in development
-    as the spec itself.  Until we have a 1.0 release
-    everything is subject to change.
-
-.. toctree::
+    There is a `tentative name change consideration
+    <https://github.com/hpk42/py-autocrypt/issues/34>`_ for this project
+    which would result in change of links.
 
-   cmdline
-   diagrams
-   api
-
-.. toctree::
-   :hidden:
+py-autocrypt provides a command line tool and a Python API to help
+mail agents integrate Autocrypt support and more.
 
-   future_work
-   pgpy_bugs
+See :doc:`install` for getting pip-installed with the ``autocrypt``
+package released on the Python Package Index.
 
-.. _installation:
+Here are some preliminary underlying aims and goals:
 
-Installation
-------------
+- `Autocrypt Level 1 compliant functionality
+  <https://github.com/hpk42/py-autocrypt/milestone/1>`_ for use by mail user agents (MUAs)
 
-You need the python package installer "pip".  If you
-don't have it you can install it on Debian systems::
+- integrate with re-mailers and other specialized server-side mail agents
```

Comment:
what do you mean by remailers? mixnet systems?

Reply:
same question, #34 says "re-mailer integration (#30)", and #30 only talks about mailman